### ISSUE TITLE
No Subtitles downloaded for Tubi.tv

### ISSUE THREAD

**Issue by sgarc103** [yt-dlp/yt-dlp#9854]
### DO NOT REMOVE OR SKIP THE ISSUE TEMPLATE

- [X] I understand that I will be **blocked** if I *intentionally* remove or skip any mandatory\* field

### Checklist

- [X] I'm reporting a new site support request
- [X] I've verified that I have **updated yt-dlp to nightly or master** ([update instructions](https://github.com/yt-dlp/yt-dlp#update-channels))
- [X] I've checked that all provided URLs are playable in a browser with the same IP and same login details
- [X] I've checked that none of provided URLs [violate any copyrights](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#is-the-website-primarily-used-for-piracy) or contain any [DRM](https://en.wikipedia.org/wiki/Digital_rights_management) to the best of my knowledge
- [X] I've searched [known issues](https://github.com/yt-dlp/yt-dlp/issues/3766) and the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues **including closed ones**. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)
- [ ] I've read about [sharing account credentials](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#are-you-willing-to-share-account-details-if-needed) and am willing to share it if required

### Region

US

### Example URLs

https://tubitv.com/movies/626964/the-nine-venoms
https://tubitv.com/movies/457303/kung-fu-of-8-drunkards
https://tubitv.com/movies/100011839/rigor-mortis-dubbed

### Provide a description that is worded well enough to be understood

yt-dlp https://tubitv.com/movies/100011839/rigor-mortis-dubbed
[TubiTv] Extracting URL: https://tubitv.com/movies/100011839/rigor-mortis-dubbed
[TubiTv] 100011839: Downloading JSON metadata
[TubiTv] 100011839: Downloading m3u8 information
[TubiTv] 100011839: Downloading MPD manifest
[TubiTv] 100011839: Downloading m3u8 information
[info] 100011839: Downloading 1 format(s): hlsv6-2504+dash-1
**[info] There are no subtitles for the requested languages**
[hlsnative] Downloading m3u8 manifest
[hlsnative] Total fragments: 1804
[download] Destination: D:\Video\Rigor Mortis (Dubbed).fhlsv6-2504.mp4
[download]   5.9% of ~   1.55GiB at    6.71MiB/s ETA 03:37 (frag 107/1804)

### Provide verbose output that clearly demonstrates the problem

- [X] Run **your** yt-dlp command with **-vU** flag added (`yt-dlp -vU <your command line>`)
- [ ] If using API, add `'verbose': True` to `YoutubeDL` params instead
- [X] Copy the WHOLE output (starting with `[debug] Command-line config`) and insert it below

### Complete Verbose Output

```shell
yt-dlp -vU https://tubitv.com/movies/100005626/invitation-only
[debug] Command-line config: ['-vU', 'https://tubitv.com/movies/100005626/invitation-only']
[debug] User config "C:\Users\sgarc\AppData\Roaming\yt-dlp\config": ['-o', 'D:\\Video\\%(title)s.%(ext)s', '--ffmpeg-location', 'C:\\yt-dlp', '--sub-format', 'srt', '--write-auto-subs', '--embed-subs', '--sub-lang', 'en.*,English', '-f', 'bestvideo[ext=mp4]+bestaudio[ext=m4a]/best[ext=mp4]/best']
[debug] Encodings: locale cp1252, fs utf-8, pref cp1252, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version stable@2024.04.09 from yt-dlp/yt-dlp [ff0779267] (win_exe)
[debug] Python 3.8.10 (CPython AMD64 64bit) - Windows-10-10.0.22000-SP0 (OpenSSL 1.1.1k  25 Mar 2021)
[debug] exe versions: ffmpeg 2023-07-19-git-efa6cec759-full_build-www.gyan.dev (setts), ffprobe 2023-07-19-git-efa6cec759-full_build-www.gyan.dev
[debug] Optional libraries: Cryptodome-3.20.0, brotli-1.1.0, certifi-2024.02.02, curl_cffi-0.5.10, mutagen-1.47.0, requests-2.31.0, sqlite3-3.35.5, urllib3-2.2.1, websockets-12.0
[debug] Proxy map: {}
[debug] Request Handlers: urllib, requests, websockets, curl_cffi
[debug] Loaded 1810 extractors
[debug] Fetching release info: https://api.github.com/repos/yt-dlp/yt-dlp/releases/latest
Latest version: stable@2024.04.09 from yt-dlp/yt-dlp
yt-dlp is up to date (stable@2024.04.09 from yt-dlp/yt-dlp)
[debug] Using fake IP 6.128.198.192 (US) as X-Forwarded-For
[TubiTv] Extracting URL: https://tubitv.com/movies/100005626/invitation-only
[TubiTv] 100005626: Downloading JSON metadata
[TubiTv] 100005626: Downloading MPD manifest
[TubiTv] 100005626: Downloading m3u8 information
[TubiTv] 100005626: Downloading m3u8 information
[debug] Formats sorted by: hasvid, ie_pref, lang, quality, res, fps, hdr:12(7), vcodec:vp9.2(10), channels, acodec, size, br, asr, proto, vext, aext, hasaud, source, id
[info] 100005626: Downloading 1 format(s): hlsv6-2563+dash-0
[info] There are no subtitles for the requested languages
[download] D:\Video\Invitation Only.mp4 has already been downloaded
[EmbedSubtitle] There aren't any subtitles to embed
```

**Comment by bashonly** [comment]
yt-dlp extracts regular subs:
```shell
$ yt-dlp --list-subs https://tubitv.com/movies/100005626/invitation-only
[TubiTv] Extracting URL: https://tubitv.com/movies/100005626/invitation-only
[TubiTv] 100005626: Downloading JSON metadata
[TubiTv] 100005626: Downloading MPD manifest
[TubiTv] 100005626: Downloading m3u8 information
[TubiTv] 100005626: Downloading m3u8 information
[info] Available subtitles for 100005626:
Language Formats
English  srt
```

The problem is that you are passing `--write-auto-subs` in your user config. Either remove that from your config, or replace it with `--write-subs`

**Comment by sgarc103** [comment]
Thank you!
